configure.ac

```diff
@@ -108,7 +108,12 @@ if test -z "$XAS"; then
         XASFLAGS="--32"
       fi
     else
-      AC_MSG_ERROR([cross-assembler not found, please install binutils-i686-linux-gnu])
+      AC_PATH_PROG([XAS], [clang])
```

Comment:
We already have that here:
https://github.com/dosemu2/dosemu2/blob/devel/configure.ac#L91
Please find why the existing fall-back
doesn't work.

Reply:
https://github.com/dosemu2/dosemu2/blob/devel/configure.ac#L110
Ah, fall-back is x86-only.
But its a bit unclear how can
clang compile non-native asm?
It seems we don't use `.arch`
directive currently.